pattern/ecs-fargate-container-image-signing-and-verification/index.md

```diff
@@ -0,0 +1,365 @@
+is running in AWS Fargate.
+:::
+
+#### Verify it works
+
+In order to launch the sample task, let's just use the Amazon ECS console for simplicity.
+
+1. Open up the Amazon ECS console and either create a new cluster, or use an existing cluster.
+2. Click to view the cluster details, and select the task list.
+3. Click the "Run new task" button.
+4. Under compute options select "Launch Type"
+5. Under deployment configuration select the family `signed-task-def`
+6. Click "Create" and observe a new task launching in the cluster
+
+Now you can visit the Lambda function logs to see whether these tasks passed or failed verification.
+
+For example here is the Lambda function output for the signed task definition:
+
+```txt
+1702417815650,"START RequestId: 25643a4a-147d-406d-bc0a-b2b5c5bae498 Version: $LATEST"
+1702417815843,"level=info msg=""Using the referrers tag schema"""
+1702417816002,"level=info msg=""Reference sha256:7ebff78b7d7bd0cb13d462ecf4d9aaa6ea7571bd5548008163d0499eae2fbf40 resolved to manifest descriptor: {MediaType:application/vnd.docker.distribution.manifest.v2+json Digest:sha256:7ebff78b7d7bd0cb13d462ecf4d9aaa6ea7571bd5548008163d0499eae2fbf40 Size:1778 URLs:[] Annotations:map[] Data:[] Platform:<nil> ArtifactType:}"""
+1702417816002,"level=info msg=""Checking whether signature verification should be skipped or not"""
+1702417816002,"level=info msg=""Trust policy configuration: &{Name:aws-signer-tp RegistryScopes:[*] SignatureVerification:{VerificationLevel:strict Override:map[]} TrustStores:[signingAuthority:aws-signer-ts] TrustedIdentities:[arn:aws:signer:us-east-2:209640446841:/signing-profiles/SigningProfile_0y9b0jhBJAoh]}"""
+1702417816002,"level=info msg=""Check over. Trust policy is not configured to skip signature verification"""
+1702417816160,"level=info msg=""Processing signature with manifest mediaType: application/vnd.oci.image.manifest.v1+json and digest: sha256:6228069a242657828cd524c21a554bb7e1f877fad63ff9c9848f3822a470a028"""
+1702417816317,"level=info msg=""Trust policy configuration: &{Name:aws-signer-tp RegistryScopes:[*] SignatureVerification:{VerificationLevel:strict Override:map[]} TrustStores:[signingAuthority:aws-signer-ts] TrustedIdentities:[arn:aws:signer:us-east-2:209640446841:/signing-profiles/SigningProfile_0y9b0jhBJAoh]}"""
+1702417816557,"Successfully verified signature for 209640446841.dkr.ecr.us-east-2.amazonaws.com/aws-signer-image-verification-registrystack-15g6wn192kvr1-signedcontainerregistry-ryrpet5bhnbg@sha256:7ebff78b7d7bd0cb13d462ecf4d9aaa6ea7571bd5548008163d0499eae2fbf40"
+1702417816561,"END RequestId: 25643a4a-147d-406d-bc0a-b2b5c5bae498"
+1702417816561,"REPORT RequestId: 25643a4a-147d-406d-bc0a-b2b5c5bae498	Duration: 910.85 ms	Billed Duration: 1308 ms	Memory Size: 1024 MB	Max Memory Used: 100 MB	Init Duration: 396.69 ms"
+```
+
+For comparison, here is the output for another unsigned image that has no signature:
```

Comment:
Could we add a output for another image which is signed but by a different signing profile